### PR TITLE
Merge in Joomla 3.9.3

### PR DESCRIPTION
**Pull request by wilsonge**
The last of Joomla 3.9.3

Merge with a merge commit. Don't rebase and merge or squash and merge

### CODE REVIEW

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 1]
```diff
@@ -98,6 +98,16 @@
 					</td>
 				</tr>
 			<?php endif; ?>
+			<?php if ($contentlang->published == -2) : ?>
+				<tr class="warning">
+					<td>
+						<span class="icon-pending" aria-hidden="true"></span><span class="element-invisible"><?php echo JText::_('WARNING'); ?></span>
```
Comment (brianteeman) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]: ``
```suggestion
						<span class="icon-pending" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
```

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 2]
```diff
@@ -98,6 +98,16 @@
...
+					</td>
+					<td>
+						<?php echo JText::sprintf('COM_LANGUAGES_MULTILANGSTATUS_ERROR_CONTENT_LANGUAGE_TRASHED', $contentlang->lang_code); ?>
```
Comment (brianteeman) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]:
```suggestion
						<?php echo Text::sprintf('COM_LANGUAGES_MULTILANGSTATUS_ERROR_CONTENT_LANGUAGE_TRASHED', $contentlang->lang_code); ?>
```

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 3]
```diff
@@ -211,11 +221,21 @@
...
-				<?php if ($status->lang_code && $status->published) : ?>
+				<?php if ($status->lang_code && $status->published == 1) : ?>
 						<td class="text-center">
 							<span class="fa fa-check" aria-hidden="true"></span>
 							<span class="sr-only"><?php echo Text::_('JYES'); ?></span>
 						</td>
+				<?php elseif ($status->lang_code && $status->published == 0) : ?>
+					<td class="text-center">
+						<span class="icon-pending" aria-hidden="true"></span>
+						<span class="sr-only"><?php echo JText::_('WARNING'); ?></span>
```
Comment (brianteeman) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]:
```suggestion
						<span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
```

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 4]
```diff
@@ -211,11 +221,21 @@
...
+					</td>
+				<?php elseif ($status->lang_code && $status->published == -2) : ?>
+					<td class="text-center">
+						<span class="icon-trash" aria-hidden="true"></span>
+						<span class="sr-only"><?php echo JText::_('WARNING'); ?></span>
```
Comment (brianteeman) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]:
```suggestion
						<span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
```

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 5]
```diff
@@ -51,6 +51,7 @@ COM_CONTACT_FIELD_CONTACT_SHOW_CATEGORY_LABEL="Contact Category"
 COM_CONTACT_FIELD_CONTACT_SHOW_LIST_LABEL="Contact Select List"
 COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL="Created By Alias"
 COM_CONTACT_FIELD_CREATED_LABEL="Created Date"
+; The following six strings are deprecated and will be removed in 4.0
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_contact.ini]: no need to merge this comment - just remove the lines ;)

File: components/com_contact/tmpl/categories/default.xml [hunk 6]
```diff
@@ -335,6 +335,20 @@
 		</fieldset>
 
 		<fieldset name="contact" label="COM_CONTACT_BASIC_OPTIONS_FIELDSET_LABEL">
+<<<<<<< HEAD:components/com_contact/tmpl/categories/default.xml
```
Comment (brianteeman) [inline components/com_contact/tmpl/categories/default.xml]: oops ;)

File: composer.json [hunk 7]
```diff
@@ -91,6 +91,7 @@
         "symfony/polyfill-php73": "^1.10",
         "symfony/web-link": "3.4.*",
         "symfony/yaml": "3.4.*",
+		"typo3/phar-stream-wrapper": "^2.0",
```
Comment (brianteeman) [inline composer.json]:
```suggestion
	"typo3/phar-stream-wrapper": "^2.0",
```

**Comment by brianteeman** [comment]
did you mean to delete  libraries/vendor/.htaccess

**Comment by brianteeman** [comment]
lol typed as you corrected

**Comment by wilsonge** [comment]
nope. that's why i already added it back :)

**Pull request merged**